### ISSUE TITLE
Resolve issue with protobuf filename registration

### ISSUE THREAD

**Issue by denalimarsh** [Kava-Labs/bep3-deputy#4]
After the changes in https://github.com/Kava-Labs/bep3-deputy/pull/2, starting the deputy gives warnings:

```
2020/05/09 00:19:14 WARNING: proto: file "abci/types/types.proto" is already registered
A future release will panic on registration conflicts. See:
https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict

2020/05/09 00:19:14 WARNING: proto: file "rpc/grpc/types.proto" is already registered
A future release will panic on registration conflicts. See:
https://developers.google.com/protocol-buffers/docs/reference/go/faq#namespace-conflict
```

Relevant snippet in the linked docs: "Warning: Retroactively changing the package name on a .proto file can potentially cause the use of extension fields or messages stored in google.protobuf.Any to stop working properly."

**Comment by fedekunze** [comment]
Probably a proto message field is using the same number for registration as another one. Are those proto files from Tendermint?

**Comment by denalimarsh** [comment]
@fedekunze yeah, we've forked `tendermint/tm-db`, `tendermint/tendermint`, `tendermint/iavl`, and `cosmos/cosmos-sdk` and updated the module path, but proto files have not been regenerated or changed in any way. The issue is probably in `tm-db` or `tendermint`.
